db/Dockerfile

```diff
@@ -8,13 +8,13 @@ ENV QC_ATLAS_CONTENT_REPOSITORY_BRANCH "master"
 ENV QC_ATLAS_SUBFOLDER_CONTENT_BACKUP_FILES "example-data/SQL/backup-files"
 ENV NISQ_CONTENT_REPOSITORY_URL "git@github.com:UST-QuAntiL/nisq-analyzer-content.git"
 ENV NISQ_CONTENT_REPOSITORY_PATH "/var/nisq/testdata"
-ENV NISQ_CONTENT_REPOSITORY_BRANCH "v1.1.0"
+ENV NISQ_CONTENT_REPOSITORY_BRANCH "master"
```

Comment:
Isn't there also a new version available?

Reply:
No, there is no newer version.

Reply:
In this case, revert this change please.